src/ui/tooltip.ts

```diff
@@ -0,0 +1,50 @@
+import { Container, Element, Label } from 'pcui';
+
+class Tooltip extends Container {
+    target: Element;
+    align: string;
+    content: Label;
+
+    constructor(args: any = {}) {
+        args = Object.assign(args, {
+            class: 'tooltip',
+            hidden: true
+        });
+
+        super(args);
```

Comment:
```suggestion
        super({
            ...args,
            class: 'tooltip',
            hidden: true
        });
```

Reply:
Since args is usually referenced after invoking `super()`, I prefer to resolve args it as it is now.

Reply:
Changed to it to use spread syntax